scripts/update-docker.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Used to restart a single msgr api container running on a server
+# First kills all running containers named msgr_api then starts a new container with the name msgr_api
+
+if [ ! -z $(docker ps -q --filter name="msgr_api") ] 
+then
+    echo "Killing all running docker processes"
+    docker kill $(docker ps -q)
```

Comment:
This is still killing all docker containers. This should be able to be changed to `docker kill msgr_api` to only kill the one container

Reply:
Good catch, I am going to change this a little bit so this command only appears in one place